aisdc/attacks/worst_case_attack.py

```diff
@@ -53,9 +62,9 @@ def __init__(self, **kwargs):
             "min_samples_leaf": 10,
             "max_depth": 5,
         }
-        self.__dict__["attack_metric_success_name"] = "AUC"
-        self.__dict__["attack_metric_success_thresh"] = 0.6
-        self.__dict__["attack_metric_success_comp_type"] = "gte"
+        self.__dict__["attack_metric_success_name"] = "P_HIGHER_AUC"
```

Comment:
I think failfast should be switched _off_ by default. It's a specific use case for researchers to avoid running lots of tests?

Comments from others? @jim-smith @rpreen @yolaj-nhs ?

Reply:
Just thinking through the logic of this...

We're implementing fail fast to act as a shortcut to a quick answer. If a user doesn't know the code very well, do we want it to run quickly or run to completion?

I'd argue we want the code to run to completion, and if they know the code well enough and know what they're looking for, they can turn on fail-fast to save themselves time

If failfast defaults to TRUE, in theory a user could run this code for their entire project and never know there's bits of information they're not seeing

That's just my thinking though - feel free to disagree!